nautobot_chatops/api/urls.py

```diff
@@ -31,11 +27,9 @@
 ]
 
 router = OrderedDefaultRouter()
-router.APIRootView = NautobotChatopsRootView
+# add the name of your api endpoint, usually hyphenated model name in plural, e.g. "my-model-classes"
 router.register("commandtoken", CommandTokenViewSet)
 router.register("accessgrant", AccessGrantViewSet)
 router.register("commandlog", CommandLogViewSet)
 
-app_name = "nautobot_chatops-api"
```

Comment:
revert?

Reply:
This is unnecessary now.

Reply:
Okay, so reading Django docs, it is possible that this is being used, but this is also missing on the Cookiecutter templates. I will open an issue and revert here and in the other apps I changed.